### PR TITLE
Hotfix assoc agg

### PR DESCRIPTION
**Pull request by aofarrel**
* Readme now says that LD prune, assoc-agg, and null model all need 2+ GDS files. I'm only certain that assoc-agg needs it, but it's likely for those other two so let's add them in just for good measure!
* assoc-agg's input checker will now ensure at least two GDS files are input, without having to localize them
* assoc-agg will no longer sneakily try to run sbg_gds_renamer before the input checker is finished
* assoc-agg **should** now be able to handle large input files (still in testing)

### CODE REVIEW

**Review by aofarrel — COMMENTED** [review]
This runs successfully on Terra, but the Cromwell timers reveal a red flag.

**2022/06/16 16:30:20** Delocalizing output /cromwell_root/glob-ce15bb98ad750635e87174959a5d6b8a -> gs://fc-secure-42a941f1-f3eb-4e10-8c47-a85916eeedfc/8f03303b-0032-4313-889d-ce5d0f6209f7/assoc_agg/fabf1432-597a-492f-a85c-3d2de7290f9a/call-sbg_prepare_segments_1/glob-ce15bb98ad750635e87174959a5d6b8a/
**2022/06/16 18:12:26** Delocalizing output /cromwell_root/glob-ce15bb98ad750635e87174959a5d6b8a.list -> gs://fc-secure-42a941f1-f3eb-4e10-8c47-a85916eeedfc/8f03303b-0032-4313-889d-ce5d0f6209f7/assoc_agg/fabf1432-597a-492f-a85c-3d2de7290f9a/call-sbg_prepare_segments_1/glob-ce15bb98ad750635e87174959a5d6b8a.list

This is for a test that resulted in 50 segments, 25 of which were a bogus chr1 that was 4.3 GB. It took almost two hours for Cromwell/Google/Terra to delocalize 25 4.3 GB zip files (the remaining zip files were less than 100 KB). It does not make intuitive sense for 107.5 GB of data to take so long to delocalize when I can download 100 GB onto my desktop computer in less than half the time.

I can't fix Cromwell/Google/Terra, though, and I don't think messing with the default number of segments is a good idea, but I can at least warn people.



TLDR -- I fixed my bug, but realized that a really big dataset might cause a thing I cannot directly control to take a very long time. Might need to step away from the zip file procedure eventually.

**Comment by aofarrel** [comment]
So it turns out throwing more money at the problem was effective. 💸 

Runtime attributes have been boosted, bringing the delocalization time down to about 30 minutes. Feeling a little more confident in this approach now, but I'd still love to get rid of the zip workaround someday.
